src/app/my-account/my-account/my-account.component.html

```diff
@@ -12,10 +12,58 @@ <h1>{{ "MY_ACCOUNT" | translate }}</h1>
     {{ "ADD_ANNOUNCEMENT" | translate }}
   </button>
 
-  <section *ngIf="myAnnouncements.length">
+  <section *ngIf="myAnnouncements?.content?.length">
     <p class="text">{{ "YOUR_ANNOUNCEMENT" | translate }}</p>
     <div class="announcements-wrapper">
-      <div *ngFor="let announcement of myAnnouncements">// TODO podpiac komponent z ogloszeniem jak bedzie gotowy</div>
+      <app-search-result
+        *ngFor="let announcement of myAnnouncements.content"
+        [location]="$any(announcement).location"
+        [origin]="$any(announcement).origin"
+        [destination]="$any(announcement).destination"
+        [title]="announcement.title"
+        [description]="announcement.description"
+        [posted]="announcement.modifiedDate"
+      >
+        <app-search-result-attribute
+          *ngIf="$any(announcement).guests"
+          [text]="
+            $any(announcement).guests + ' ' + ($any(announcement).guests | pluralizePolish: 'osoba':'osoby':'osób')
```

Comment:
'osoba':'osoby':'osób' - to powinno przejść przez pipe translate z odpowiednimi kluczami

Reply:
Tak, tylko co z innymi wersjami językowymi tam inna odmiana? Może lepiej jakoś inaczej prezentować tą informację żeby nie było potrzeby odmiany?

Reply:
odmiana powinna być obsłużona przez narzędzie do i18n, tylko że takiego nie używamy...

Reply:
@bbartosiak czekamy na aktualizacje makiet czy mamy ten tekst samodzielnie zastąpić jakimś innym?

Reply:
Zostało ustalone ze zmieniamy na tekst: Liczba osób: <liczba>. Cezary poprawi u siebie, skontaktuj się z nim

Reply:
Już poprawiłem.